### ISSUE TITLE
Running migrate twice gives an error about changed models

### ISSUE THREAD

**Issue by dwaynebailey** [translate/pootle#3687]
If you run ``migrate`` a second time directly after an initial migration you will get the following error.
```
Running migrations:
  No migrations to apply.
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

``makemigrations`` produces this file:

```py
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import models, migrations
import pootle.core.markup.fields


class Migration(migrations.Migration):

    dependencies = [
        ('virtualfolder', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='virtualfolder',
            name='description',
            field=pootle.core.markup.fields.MarkupField(help_text='Use this to provide more information or instructions. Allowed markup: HTML', verbose_name='Description', blank=True),
            preserve_default=True,
        ),
    ]
```

@unho Why are virtualfolders doing this?

**Comment by julen** [comment]
That's because `MarkupField`'s `help_text` has a placeholder in it, and the initial migration contains `reStructuredText` as the value for the placeholder, whereas in your case the value is `HTML`. A similar thing happened with #3557.

**Comment by unho** [comment]
I understand the problem, but I am not sure how to fix it. Subclassing `MarkupField` would require doing the same thing in all the other places where it is used.

**Comment by dwaynebailey** [comment]
@unho if that's what's required then yes.  It's more of an issue that post migration if you have ``MARKUP_FILTER`` set to anything but rST then you find migrations are now out of sync.  I guess its a bad side effect of ``MarkupField``

**Comment by julen** [comment]
It's not `MarkupField`'s side-effect but dynamic help text's. The same can happen for any field with dynamic values which are evaluated when models are loaded.

**Comment by unho** [comment]
Just merged commit b77fef61 which fixes this issue when no markup is defined in the settings (thus defaulting to HTML).

But please note that if the markup is changed in the settings, a similar message asking for creating migrations is shown, and this time it creates several new migrations. So I don't think we can consider this issue to be fixed.
